src/lib/api.js

```diff
@@ -38,19 +38,21 @@ function setupDOM(handler) {
 }
 
 function onReset(handler) {
-  handler.ptrElement.classList.remove(`${handler.classPrefix}refresh`);
-  handler.ptrElement.style[handler.cssProp] = '0px';
-
-  setTimeout(() => {
-    // remove previous ptr-element from DOM
-    if (handler.ptrElement && handler.ptrElement.parentNode) {
-      handler.ptrElement.parentNode.removeChild(handler.ptrElement);
-      handler.ptrElement = null;
-    }
-
-    // reset state
-    _shared.state = 'pending';
-  }, handler.refreshTimeout);
+  if(handler.ptrElement) {
```

Comment:
Please rebase from master and run `make` to run tests locally, or just wait for github-actions to complete.

Thank you!

Reply:
Yeah good idea. Do whatever you want, this project is yours :-) I’m used to keep code as readable as possible and to avoid multiple returns in functions, but I’m not a maintainer of this project, I just made this change at your request to fix the bug I reported. Merry Christmas!